route/build/src/histVars_data.f90

```diff
@@ -216,14 +228,18 @@ SUBROUTINE aggregate(this,             & ! inout:
 
         do ix=1,this%nRch
           this%discharge(ix,iRoute) = this%discharge(ix,iRoute) + RCHFLX_local(1,ix)%ROUTE(idxMethod)%REACH_Q
-          this%volume(ix,iRoute)    = this%volume(ix,iRoute) + RCHFLX_local(1,ix)%ROUTE(idxMethod)%REACH_VOL(1)
+          this%volume(ix,iRoute)    = RCHFLX_local(1,ix)%ROUTE(idxMethod)%REACH_VOL(1)
```

Comment:
Looks like this has lots of uses of "1". This might be out of scope for this PR, but it would be good to have a parameter that has meaning in it's name here.

Reply:
0 and 1 will be replaced with parameters. This change (issue #532) will be made in another PR.